portfolio/src/components/Input.tsx

```diff
@@ -0,0 +1,15 @@
+import clsx from "clsx";
+import type { FC, InputHTMLAttributes } from "react";
+import { inputClassName } from "../constants";
+
```

Comment:
Here's another custom component that I made. There's not as much utility in having this be a custom component as there is in the custom `Button` component. But it was just another case of me seeing that there were a bunch of the same `classNames` being applied across several instances of the `input` tag. So just figured it would prevent duplicate code and would benefit you later to just have a single place where you can define properties to be shared across all of the various `input`s in the project